src/test/scala/is/hail/TestUtils.scala

```diff
@@ -256,34 +247,11 @@ object TestUtils {
       .exportGen(path, precision)
   }
 
-  def assertEvalsTo(x: IR, expected: Any) {
-    assertEvalsTo(x, Env.empty, FastIndexedSeq(), None, expected)
-  }
-
-  def assertEvalsTo(x: IR, agg: (IndexedSeq[Row], TStruct), expected: Any) {
-    assertEvalsTo(x, Env.empty, FastIndexedSeq(), Some(agg), expected)
-  }
-
-  def assertEvalsTo(x: IR, env: Env[(Any, Type)], args: IndexedSeq[(Any, Type)], agg: Option[(IndexedSeq[Row], TStruct)], expected: Any) {
-    val t = x.typ
-    assert(t.typeCheck(expected))
-
-    val e = eval(x, env, args, agg)
-    assert(t.typeCheck(e))
-    assert(t.valuesSimilar(eval(x, env, args, agg), expected))
-  }
-
   def eval(x: IR): Any = eval(x, Env.empty, FastIndexedSeq(), None)
 
   def eval(x: IR, agg: (IndexedSeq[Row], TStruct)): Any = eval(x, Env.empty, FastIndexedSeq(), Some(agg))
 
   def eval(x: IR, env: Env[(Any, Type)], args: IndexedSeq[(Any, Type)], agg: Option[(IndexedSeq[Row], TStruct)]): Any = {
-    val i = Interpret[Any](x, env, args, agg)
```

Comment:
I think this change probably requires that `InterpretSuite` call a different method, since right now it's using `eval` to check that the compiler and interpreter agree. (and now I think it would just be checking that the expression compiles and doesn't error?)

Reply:
Whoops, I forgot to check what else was using eval.